promscale/installation/docker.md

```diff
@@ -93,17 +97,29 @@ You can also migrate to Debian version by doing the following (please note: this
      END LOOP;
    END$$;
    ```
+   This is necessary because the collation in the Alpine image is broken and so
+   BTREE-based indexes remain incorrect until they are reindexed. It is
+   extremely important to execute this step before ingesting new data to avoid
+   data corruption. This process can take a long time depending on the indexed
+   textual data in the database.  
+
+1. Restart the Promscale Connector
 
-7. Restart the Promscale Connector
+</procedure>
+
+If you are using Kubernetes instead of plain docker:
+<procedure>
 
-If you are using Kubernetes instead of plain docker you should:
 1. Shutdown the Promscale Connector pods
-2. Change the database pod to use the debian docker image and restart it.
-3. Execute jobs for the script in steps 4 and 6 above.
-4. Restart the Promscale Connector pods.
+1. Change the database pod to use the debian docker image and restart it.
+1. Execute jobs for the script in steps 4 and 6 above.
```

Comment:
```suggestion
1. Execute jobs for the script in steps 4 and 6 above.
```

Actually type out the steps here, don't reference others. You don't know if readers have seen the previous procedure, and you can't guarantee that steps 4 and 6 will remain the same over time.

Reply:
Hi @VineethReddy02  for now I have copied the steps 4 and 6 to the Kubernetes section. However, the commands will be different in Kubernetes. Please provide the commands.